kyaml/runfn/runfn.go

```diff
@@ -57,6 +58,12 @@ type RunFns struct {
 	// and only use explicit sources
 	NoFunctionsFromInput *bool
 
+	// EnableStarlark will enable functions run as starlark scripts
+	EnableStarlark bool
+
+	// DisableContainers will disable functions run as containers
+	DisableContainers bool
```

Comment:
This is a boolean mess.

Reply:
can we do an enumeration of legal states instead?

Reply:
or are all four states legal?  do we add more booleans for _cue_ or whatever's next?  
I've read the PR description, but it's not helping me.

Reply:
- yes all 4 states are legal -- you can enable starlark only, containers only, starlark and containers, neither starlark or containers.  The last state is legal but not useful.
- booleans map pretty well to command line flags, enumerations less so
- most folks should be using the defaults

This api doesn't have a v1.0.0 release, so it is pretty easy to change.  I'd suggest we keep this as is and refactor when it is more obvious what a better representation would be.